stabilizer/src/main/java/com/hazelcast/stabilizer/agent/workerjvm/WorkerJvmLauncher.java

```diff
@@ -124,10 +130,27 @@ private WorkerJvm startWorkerJvm(String mode) throws IOException {
         new WorkerJvmProcessOutputGobbler(process.getInputStream(), new FileOutputStream(logFile)).start();
         workerJvm.process = process;
         workerJvm.mode = WorkerJvm.Mode.valueOf(mode.toUpperCase());
+        copyResourcesToWorkerId(workerId);
         workerJVMs.put(workerId, workerJvm);
         return workerJvm;
     }
 
+    private void copyResourcesToWorkerId(String workerId) throws IOException {
+        final String testSuiteId = agent.getTestSuite().id;
+        if (!new File(WORKERS_PATH + testSuiteId + "/upload/").exists()) {
```

Comment:
first version of this block has a true file path but i removed based on review this block. When i added again i made a copy paste mistake. First version (one withe the true path)  tested real enviroment.